packages/mds-ingest-service/repository/migrations/1603127449428-InitialMigration.ts

```diff
@@ -0,0 +1,35 @@
+import { MigrationInterface, QueryRunner } from 'typeorm'
+
+export class InitialMigration1603127449428 implements MigrationInterface {
+  name = 'InitialMigration1603127449428'
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
```

Comment:
You need to add a bypass valve to `up` for existing tables (See mds-attachment-service). Also, I wouldn't mind seeing separate `Create{Devices/Events/Telemetry}Table` migrations.